tests/test_contrib_aiohttp.py

```diff
@@ -150,3 +151,17 @@ class TestAIOHTTP:
             await wait_for(collected)
 
         assert collected == []
+
+    async def test_client_closed_correctly(self):
```

Comment:
Better to improve test and some timeout after which it fails, it's not good to hang, for example, on CI.